src/localization/locales/en-US.json

```diff
@@ -279,7 +279,8 @@
         "description_update": "Update description",
         "delete_label": "Delete “{{name}}”",
         "download_label": "Download “{{name}}”",
-        "invalid_extension": "The file extension ({{file.resourceType}}) does not match the file's contents"
+        "invalid_extension": "The file extension ({{file.resourceType}}) does not match the file's contents",
+        "loader_label": "[TODO] Loading files"
```

Comment:
@paulschreiber Paul please check this string, is the separate loader for shared data entries

Reply:
`Loading…` is fine.